npm/webpack-preprocessor/package.json

```diff
@@ -22,7 +22,7 @@
   "dependencies": {
     "bluebird": "3.7.1",
     "debug": "^4.3.2",
-    "del": "^6.1.1",
+    "fs-extra": "^10.1.0",
```

Comment:
Lets remove `"fs-extra": "9.1.0",` from the devDependencies list

Reply:
Good call. Addressed in https://github.com/cypress-io/cypress/pull/23283/commits/6bea1fc483c4228c4c11bd4d02585576316fed90